client/my-sites/plans-comparison/plans-comparison-features.tsx

```diff
@@ -120,7 +120,9 @@ export const planComparisonFeatures: PlanComparisonFeature[] = [
 
 			return feature
 				? translate( 'Custom domain name is free for one year!' )
-				: translate( 'Custom domain name is not included' );
+				: translate( 'Custom domain name is {{strong}}not{{/strong}} included', {
```

Comment:
@niranjan-uma-shankar will this create a new call for translations? Any thoughts to get around if so, as it's only stylistic changes? :-)

Reply:
> stylistic changes

in `en` at least 🤔

Reply:
> stylistic changes

in `en` at least 🤔

Reply:
I believe this will only result in some fuzzy strings which can be resolved relatively easy. We can also add "String Freeze" label to see what will happen :)

Reply:
TIL about `String Freeze`. I may not recall correctly, but I think we merged stuff early behind features flags in the past to get translations before rollout, but this is obviously a solver (if I understand).

cc @razvanpapadopol 😎

Reply:
`[Status] String Freeze` label is used for Calypso changes where you want to have the strings translated before you merge the PR. Otherwise, they will be sent for translation anyway when you merge the PR but you should have some mechanism in place to prevent showing the untranslated strings to real users. Read more in FG: PCYsg-3hI-p2#calypso

> we merged stuff early behind features flags in the past to get translations before rollout

This is the only way for ETK, where the process starts only after a version of the plugin containing those strings is released. Read more about ETK processs: PCYsg-ly5-p2#i18n

Reply:
Ah, yes. Indeed recall the ETK scenario now. So yeah, different process. Thanks for the insight/summary 😎